core/src/main/java/org/aerogear/mobile/core/metrics/MetricsService.java

```diff
@@ -0,0 +1,59 @@
+package org.aerogear.mobile.core.metrics;
+
+import org.aerogear.mobile.core.MobileCore;
+import org.aerogear.mobile.core.ServiceModule;
+import org.aerogear.mobile.core.configuration.ServiceConfiguration;
+import org.aerogear.mobile.core.metrics.interfaces.MetricsPublisher;
+import org.aerogear.mobile.core.metrics.interfaces.Observer;
+
+public class MetricsService implements ServiceModule {
```

Comment:
[Optional] If something is in the core it may not need that interface. Not sure if that should be actual service if that's part of the core.

Reply:
@wtrocki I guess the nice thing with using that interface is that this way it behaves just like any other SDK module and `core.getInstance` can be used to retrieve the module instead us cluttering `MobileCore` with another `getXXXService` method. Sure, i added a convenience method to core to retrieve a metrics publisher but we could just as well remove it.